packages/client/src/components/Select/CustomSelect.jsx

```diff
@@ -3,7 +3,12 @@ import { FormControl, InputLabel, Select, MenuItem } from '@mui/material';
 import { useMovieList } from '../../context/movieListContext';
 
 const CustomSelect = () => {
-  const { sortBy, setSortBy } = useMovieList();
+  const { sortBy, setSortBy, isClickedSame, setIsClickedSame } = useMovieList();
+  const handleMenuItemClick = (e) => {
+    e.target.id === sortBy
+      ? setIsClickedSame(!isClickedSame)
```

Comment:
I normally suggest to toggle state by reference to previous value inside the function like so:
`setIsClickedSame(prevState => !prevState)` so it is safer